src/renderer/components/SyncView/SyncAppBar.js

```diff
@@ -1,15 +1,29 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import AppBar from '@material-ui/core/AppBar'
 import Toolbar from '@material-ui/core/Toolbar'
 import Typography from '@material-ui/core/Typography'
 import Button from '@material-ui/core/Button'
 import { makeStyles } from '@material-ui/core/styles'
+import { Wifi, WifiOff } from '@material-ui/icons'
 import { defineMessages, useIntl } from 'react-intl'
+import wifi from 'node-wifi'
+import { Tooltip } from '@material-ui/core'
+
+wifi.init({
```

Comment:
Is this OK to be doing this in the renderer process? I wonder if it will visibly block it from refreshing?

We could let this go in to master for now and I can rebase it onto my branch for putting a lot of the code into a background IPC process (#259) which would ensure it would not block the renderer. This would require line 40 to be using the ipc to fetch the current connections, but otherwise the code shouldn't have to change.

Reply:
I can't see why any of these methods would be processor intensive, so I don't think they would affect the renderer.

Reply:
The init function seems to just be doing some options initialisation and a bit of currying : the functions like `linuxConnect` in the `switch/case` have a prototype like `(config) => () => {}` so they don't execute anything.

Reply:
https://github.com/friedrith/node-wifi/blob/master/src/wifi.js#L20